app/src/index.html

```diff
@@ -6,8 +6,15 @@
   name="viewport"
   content="minimum-scale=1, initial-scale=1, width=device-width"
 />
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet.draw/1.0.4/leaflet.draw.css"/>
```

Comment:
We should add the styling the same way as the other css files. See styles.css for examples.